dataeng/jobs/analytics/SqlScripts.groovy

```diff
@@ -37,6 +37,13 @@ class SqlScripts {
 
     public static def single_script_job = { dslFactory, allVars ->
         dslFactory.job("single-sql-script") {
+            authorization {
+                allVars.get('USER_ROLES').each { github_id, roles ->
+                    roles.each {
+                        permission(it, github_id)     // it is an implicit parameter corresponding to the current element
```

Comment:
Nit: can't you just do
```
roles.each { role ->
    permission(role, github_id)
```
to be explicit?  That's also more consistent with at least the DSL that I've been writing.

Reply:
I'll try that.